src/Flex.php

```diff
@@ -299,7 +300,7 @@ public function install(Event $event)
         $this->update($event);
     }
 
-    public function update(Event $event, $operations = [])
+    public function update(UpdateEvent $event, $operations = [])
```

Comment:
This should be reverted: Composer itself calls update() when triggering events and doesn't know about UpdateEvent.